pyqg/model.py

```diff
@@ -19,7 +21,26 @@
 except ImportError:
     pass
 
-class Model(PseudoSpectralKernel):
+kernel_attrs = [
+    'nx', 'ny', 'nz', 'nl', 'nk', 'a', 'kk', 'll', 
+    'q', 'qh', 'ph', 'u', 'v',
+    'Ubg', 'Qy', 'ufull', 'vfull',
+    'uh', 'vh', 'rek', 't', 'tc', 'dt',
+    'uv_parameterization', 'q_parameterization',
+     'fft', 'ifft', 'filtr', 'ablevel',
+    'dqdt', 'dqhdt', 'dqhdt_p', 'dqhdt_pp', 'dqh', 'duh', 'dvh',
+    'uq', 'vq',
+    '_invert',
+    '_forward_timestep',
+    '_do_advection',
+    '_do_friction',
+    '_do_q_subgrid_parameterization',
+    '_do_uv_subgrid_parameterization',
+    '_ik', '_il',
+]
+
+@delegate(*kernel_attrs, to='kernel')
```

Comment:
So I think the delegation here is good, but any thoughts on how to organize the list of attributes we're delegating (or if there are any we should remove the need for)?

Reply:
There are fewer attributes being delegated now, so maybe this is ok :)